python/tests/README.md

```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
    `./mvnw quarkus:dev -am -pl :nessie-quarkus` locally.
 1. Run `pytest --record-mode=all tests/` (in a venv with `requirements_dev.txt` installed)
    to add test data for newly added tests and updated tests data.
+   * Note: when running `test_nessie_auth.py` in recording more, the tests will most likely fail out-of-the box
+     because the server will not recognize the auth tokens those tests use. In order to record cassettes for the
+     authentication tests:
+     1. Configure the Nessie server in dev mode with a real authentication provider (e.g. Keycloak)
+     1. Generate real auth tokens
+     1. Put real tokens into test sources
```

Comment:
Let's mention the parameter in application.properties.

Could we add one as a comment there?

Reply:
I'm not sure what you mean, sorry. The token is not meant to be in application.properties at all

Reply:
Strike my comment - it wasn't right.